ert3/evaluator/__init__.py

```diff
@@ -0,0 +1,9 @@
+def evaluator(coefficients):
```

Comment:
As the function carries out an action, should we change its name to `run`, `evaluate` or similar?

Reply:
I agree!